Example/CardScan/ViewController.swift

```diff
@@ -208,13 +216,12 @@ extension ViewController: SimpleScanDelegate {
     }
 }
 
-extension ViewController: ScanEvents {
-    func onNumberRecognized(number: String, expiry: Expiry?, numberBoundingBox: CGRect,
-                            expiryBoundingBox: CGRect?, croppedCardSize: CGSize, squareCardImage:
-                                CGImage, fullCardImage: CGImage, centeredCardState: CenteredCardState?, uxFrameConfidenceValues: UxFrameConfidenceValues?, flashForcedOn: Bool) {}
+extension ViewController: ScanEventsDelegate {
+    func onNumberRecognized(number: String, expiry: Expiry, numberBoundingBox: CGRect, expiryBoundingBox: CGRect, croppedCardSize: CGSize, squareCardImage: CGImage, fullCardImage: CGImage) {}
+    
     func onScanComplete(scanStats: ScanStats) {}
-    func onFrameDetected(croppedCardSize: CGSize, squareCardImage: CGImage, fullCardImage: CGImage,
-                         centeredCardState: CenteredCardState?, uxFrameConfidenceValues: UxFrameConfidenceValues?, flashForcedOn: Bool) {}
+    
+    func onFrameDetected(croppedCardSize: CGSize, squareCardImage: CGImage, fullCardImage: CGImage) {}
```

Comment:
We need these parameters for verify. I don't think we can safely remove them here.

Reply:
oh i just replaced that in the example to see if the new delegate worked. With `ScanEventsDelegate` (the objc protocol I made for verif-y), I removed all the `cardverify` related parameters since they're just using `cardscan`

Reply:
ahh I see, I misread this. thanks.